plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/TrinoS3FileSystem.java

```diff
@@ -196,6 +199,9 @@
     public static final String S3_STREAMING_UPLOAD_PART_SIZE = "trino.s3.streaming.part-size";
     public static final String S3_STORAGE_CLASS = "trino.s3.storage-class";
 
+    public static final String STS_ENDPOINT = "trino.sts.endpoint";
+    public static final String STS_REGION = "trino.sts.region";
```

Comment:
As these are used only for accessing S3 I would use `trino.s3.sts.endpoint/region`. And S3_STS_* as constant names.

Reply:
Makes sense to me, will update as soon as I know how to proceed regarding your first comment.